cardano_cli_helper.py

```diff
@@ -70,7 +70,11 @@ def getAddrUTxOs(addr, network='mainnet'):
         utxosJson = json.load(file)
         file.close()
         os.remove(outfile)
-        return utxosJson
+        if onlyAda:
+            # only return utxo if they contain one token type (ie lovelace)
+            return { utxo: utxosJson[utxo] for utxo in utxosJson if len(utxosJson[utxo]['value'].keys()) == 1}
+        else:
+            return utxosJson
```

Comment:
I found it simpler to handle the utxo with native tokens by just filtering them out instead of trying to balance them.

Reply:
Sounds good